_docs/info/getstarted.md

```diff
@@ -92,7 +92,35 @@ developer/gcc5                                    5.5.0-0.151024             ---
 developer/gcc6                                    6.4.0-0.151024             ---
 ```
 
-and then install the desired version along with the `system/header` package:
+Note that our query is run against the **entire package name**, including its category (in this case, developer), however a short form can be used. For example, `bhyve` would match against `system/bhyve`, `system/bhyve/firmware`, `system/library/bhyve`, and `system/zones/brand/bhyve`, but `gcc` would not match against `gcc5` due to the extra `5`. Globbing is provided (remember to use quotes if using e.g. bash), using the `?` and `*`  wildcards to match one or zero or more instances of any character respectively. Therefore, for the most general search for a term, use the following:
```

Comment:
The short form only matches the trailing part, so `bhyve` does not match `system/bhyve/firmware`, although `firmware` would and so would `bhyve/firmware`.

I think you can drop the `if using e.g. bash` bit - if the command is being used interactively, then the quotes are going to be necessary regardless of what shell is in use.

Reply:
I actually ran that query for `bhyve` and copied the output in - I'll run it again and double-check and then edit.